### PR TITLE
[DNM] Move status filter out of ListView

### LINKED ISSUE #227: Move status filter out of ListView
The task status filter is currently implemented within ListView.js.  It should be moved into TaskList.js as it is a task-specific feature and belongs there.

### PR DESCRIPTION
**Pull request by DavisRayM**
- [x] Remove Status Filter From ListView
- [x] Add Status Filter to ListView

Fix #227

### CODE REVIEW

**Comment by moshthepitt** [comment]
@DavisRayM is this ready for review?

File: src/containers/ElementMap.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 // This component takes a list of strings
-// and returns them  wrapped in the provided HTML tag
+// and returns them wrapped in the provided HTML tag
 import React, { Component } from "react";
+import { DropdownToggle, DropdownMenu, Dropdown } from "reactstrap";
+import PropTypes from "prop-types";
 
 export default class ElementMap extends Component {
+  static propTypes = {
+    filterFields: PropTypes.object
```
[the remark this answers is not in the document]
Comment (DavisRayM) [reply src/containers/ElementMap.js]: Will do so.

File: src/containers/ElementMap.js [hunk 2]
```diff
@@ -14,6 +20,40 @@ export default class ElementMap extends Component {
         [item]
       )
     );
+
+    if (this.props.filterFields != null) {
```
[the remark this answers is not in the document]
Comment (DavisRayM) [reply src/containers/ElementMap.js]: Makes sense. Will create a new component to hold the function in my upcoming pushes.

**Comment by moshthepitt** [comment]
@DavisRayM I wanted to make some related commits and realised that I needed to create a new branch.  Moved your PR here https://github.com/onaio/kaznet-frontend/pull/234.  Will close this one and merge that one.  Thanks for the excellent work!

**Pull request closed without merging**